core/scripts/Voting.js

```diff
@@ -522,7 +533,7 @@ class VotingSystem {
       return await _constructReveal(request, roundId, web3, this.account, this.voting);
```

Comment:
@ptare I think this call `_constructReveal` relies on the old EncryptedStore

Reply:
@nicholaspai This PR also modifies VotingUtils.js

Reply:
good point

Reply:
Which I think the CLI also uses, so the fix to the CLI might not be that bad.

Reply:
The dapp, on the other hand, is gonna be annoying lol.

Reply:
> Which I think the CLI also uses, so the fix to the CLI might not be that bad.

correct.